content/ja/docs/concepts/services-networking/service.md

```diff
@@ -23,7 +23,7 @@ KubernetesはPodにそれぞれのIPアドレス割り振りや、Podのセッ
 
 ## Serviceを利用する動機
 
-{{< glossary_tooltip term_id="pod" text="Pod" >}}は停止が想定して設計されています。 Podが作成され、もしそれらが停止する時、Podは再作成されません。
+Kubernetes {{< glossary_tooltip term_id="pod" text="Pods" >}}はクラスターの状態に合わせて作成され削除されます。Podは揮発的なリソースです。
```

Comment:
As described in https://kubernetes.io/ja/docs/contribute/localization/, please do not use plural for resource names.

Reply:
The original text says "Pods", but you mean i need rephrase to "Pod"?
Some languages expresses "Pods" in the docs.
https://v1-19.docs.kubernetes.io/zh/docs/concepts/services-networking/service/

Reply:
But, yes, if I need to fix the expression, how about this amendment? 👀

```suggestion
Kubernetes {{< glossary_tooltip term_id="pod" text="Pod" >}}はクラスターの状態に合わせて作成され削除されます。Podは揮発的なリソースです。
```

Reply:
![image](https://user-images.githubusercontent.com/20236173/111055891-9d704c80-84bd-11eb-890c-1cd6d32e2215.png)

Reply:
@inductor Ok,  i have applied the suggestion! could you check it?